src/Microsoft.Azure.WebJobs.Host/Executors/IFunctionInvoker.cs

```diff
@@ -1,16 +1,23 @@
 ﻿// Copyright (c) .NET Foundation. All rights reserved.
 // Licensed under the MIT License. See License.txt in the project root for license information.
 
-using System.Collections.Generic;
 using System.Threading.Tasks;
 
-namespace Microsoft.Azure.WebJobs.Host.Executors
+namespace Microsoft.Azure.WebJobs.Host
 {
-    internal interface IFunctionInvoker
+    /// <summary>
+    /// Provides an interface for invoking a job function.
+    /// </summary>
+    public interface IFunctionInvoker
```

Comment:
Making this public. We'll also be able to replace https://github.com/Azure/azure-webjobs-sdk-script/blob/1ab54a94916c8180105af1e43b24a483f070a53f/src/WebJobs.Script/Description/IFunctionInvoker.cs#L9 with this.